msteams-platform/sbs-botbuilder-linkunfurling.yml

```diff
@@ -99,7 +99,7 @@ items:
 
     1. Select **Review + create**.
 
-       :::image type="content" source="~/assets/images/meeting-token-generator/register-bot-3.png" alt-text="Create Microsoft App ID":::
+       :::image type="content" source="~/assets/images/Tab-ME-SSO/register-bot-2.png" alt-text="Create Microsoft App ID":::
```

Comment:
The alt-text ends with a period. Check the best practices for alt-text in the Contributor's guide.

Reply:
updating alt text in different PR